client/src/app/api/rest.ts

```diff
@@ -543,6 +545,17 @@ export const removeFileTaskgroup = ({
   );
 };
 
+export const getMigrationWaves = (): Promise<MigrationWave[]> =>
+  axios.get(MIGRATION_WAVES).then((response) => response.data);
+
+export const createMigrationWave = (
+  obj: MigrationWave
+): Promise<MigrationWave> =>
+  axios.post(MIGRATION_WAVES, obj).then((response) => response.data);
+
```

Comment:
If you replace the request without the response then all the Success/Error notifications will work:
```
export const createMigrationWave = (
  obj: MigrationWave
): Promise<MigrationWave> => axios.post(MIGRATION_WAVES, obj);
```